### PR TITLE
Update minimum PHP version for Cake 5

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

File: config/requirements.php [hunk 1]
```diff
@@ -20,8 +20,8 @@
 /*
  * You can remove this if you are confident that your PHP version is sufficient.
  */
-if (version_compare(PHP_VERSION, '7.2.0') < 0) {
-    trigger_error('Your PHP version must be equal or higher than 7.2.0 to use CakePHP.', E_USER_ERROR);
+if (version_compare(PHP_VERSION, '8.1.0', '<')) {
```
Comment (ADmad) [inline config/requirements.php:23]: I don't remember why we needed this, that's composer's job.

Comment (dereuromark) [reply config/requirements.php:23]: Wasnt there still a way to install without composer or alike?

Comment (garas) [reply config/requirements.php:23]: Composer added this check in 2.0.0.

Comment (ADmad) [reply config/requirements.php:23]: > Wasnt there still a way to install without composer or alike?

We do or did full zipballs. I personally have no interest in catering for that tiny fraction of users.

Comment (garas) [reply config/requirements.php:23]: Zipballs already include vendor/ with Composer PHP version check.

**Pull request merged**